docs/blog/20231109-Daeraxa-NovemberUpdate.md

```diff
@@ -0,0 +1,87 @@
+- Likewise, there is less overhead associated with keeping things up-to-date, and it will be easier to update when needed.
+- Improvements added to the community package READMEs can be used everywhere, not just on the Pulsar Package Website.
+- We could introduce global Pulsar settings for greater control over how markdown is parsed, which can apply to every package that uses this API. For example, enabling emojis, adding copy buttons to code blocks, disabling syntax highlighting and modifying links or images.
+- Automatic redirection of "dead" `atom.io` URLs to the `pulsar-edit.dev` or web archive equivalent.
+
+Of course this new API will also be available for use by community packages for developers to take advantage of if they wish, so that Markdown parsing doesn't need to be independently implemented any more.
+
+This new feature is availble in our newest rolling releases and will soon be in our regular release too!
+
+## Exposing a `fuzzy-native` API
+
+<img src="./assets/detective.png" height=200>
+
+In a similar vein to the above, we also have a [fuzzy-native](https://github.com/pulsar-edit/fuzzy-native) package used to provide fuzzy string matching for Pulsar. This package is used as a dependency for a large number of Pulsar packages, including core packages such as `fuzzy-finder`, `command-palette` and `autocomplete-plus`. The goals here are very similar to those for the new UI API mentioned above; instead of each package needing to specify (and bundle) `fuzzy-native` as a dependency, they can instead simply reference this new global API, saving the need to bundle it as a dependency and having to maintain each package that uses it.
+
+You can view the progress of this work over on [its pull request](https://github.com/pulsar-edit/pulsar/pull/774) by [@mauricio szabo].
+
+## A new code runner package
+
+<img src="./assets/computer.png" height=200>
+
+[@confused-techie] has been working on a new package for running scripts and code directly within Pulsar. The inspiration comes from some existing packages that still work and can be installed, but few of the most popular ones are still actively maintained. Having an integrated code runner package is a common request from new Pulsar users wondering how they can run code directly in the editor in much the same way as is possible within other editors such as VSCode. We are currently discussing the best way to bring this kind of functionality to users; should we publish this as a package, include it as a new core package, or something in-between? Whatever the outcome, you can look forward to more news on this package soon!
+
+If you want to check it out in its current, in-development state, then have a look at the [pulsar-runner](https://github.com/confused-Techie/pulsar-runner) repository.
+
+<img src="./assets/pulsar-runner.png" height=200>
+
+## Further instalments in the modern Tree-sitter blog post series
```

Comment:
```suggestion
## Further installments in the modern Tree-sitter blog post series
```

Reply:
Classic UK vs US spelling again, I guess it should be US as the blog is meant to be from the team.

Reply:
TIL This is the UK spelling, was throwing me off why it didn't show as wrong on autocorrect lol